src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Fragment } from 'react';
+
+import { parseHtml } from 'utils/parse-html';
+
+export type DescriptionProps = {
+  title: string;
+  markup: string;
+  url: string;
+  name: string;
+};
+
+type DescriptionListProps = {
+  items: DescriptionProps[];
+  id: string;
+};
+
+export const DescriptionList = ({ items, id }: DescriptionListProps) => (
+  <div className="cc-description-list">
+    <dl>
+      {items
+        .filter(({ markup }) => markup !== null)
```

Comment:
If markup can be (typeof) `null` then it should be marked as optional in the relevant type alias.

```
markup?: string
```

Also might be worth thinking about what to do if the string is empty (rather than null).